### ISSUE TITLE
skeleton loaders

### ISSUE THREAD

**Issue by ThaUnknown** [halfmoonui/halfmoon#45]
a class for skeleton loaders based on background color and text color would be really cool, not many frameworks include skeleton loaders

**Comment by halfmoonui** [comment]
I am not entirely sure about this as a component, but I will still mark this.

**Comment by ThaUnknown** [comment]
> I am not entirely sure about this as a component, but I will still mark this.

I don't know if it would be possible to have one universal component for all of them since for example cards, headers, text, backgrounds etc would probably all need their own skeleton loader, and it might be quite hard for people to use it, but IMO it's defo worth trying to do

**Comment by halfmoonui** [comment]
Exactly. This is the issue with skeleton loaders and why I said I am not sure about adding one.
